recipe/run_test.sh

```diff
@@ -3,7 +3,24 @@ system=$(uname -s)
 
 case $system in
 	"Linux" )
-		gcc igraph_test.c $(pkg-config --libs --cflags igraph) -o igraph_test
+        export CC=gcc
+		${CC} igraph_test.c $(pkg-config --libs --cflags igraph) -o igraph_test
 		./igraph_test
 		;;
+    "Darwin" )
+        export CC=clang
+        export CXX=clang++
+        export MACOSX_VERSION_MIN="10.7"
+        export MACOSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+        export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+        export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+        export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+        export LDFLAGS="${LDFLAGS} -stdlib=libc++ -lc++"
+        export LINKFLAGS="${LDFLAGS}"
```

Comment:
Adding the `toolchain` package to `build` does this for you. So you can drop this stuff.